plugins/digitalocean/resoto_plugin_digitalocean/resources.py

```diff
@@ -61,14 +64,55 @@ def delete(self, graph: Graph) -> bool:
         raise NotImplementedError
 
     def update_tag(self, key, value) -> bool:
-        """Update a resource tag in the cloud"""
-        log.debug(f"Updating or setting tag {key}: {value} on resource {self.id}")
-        return True
+
+        if self.tag_resource_name():
```

Comment:
Consider to add a note that not all resources allow tagging.

Reply:
Marked as resolved, but I do not see a note?

Reply:
Will add it in the next PR.